ScoutSuite/output/data/html/partials/azure/services.keyvault.vaults.html

```diff
@@ -0,0 +1,25 @@
+<!-- Key Vault partial -->
+<script id="services.keyvault.vaults.partial" type="text/x-handlebars-template">
+    <div class="list-group-item active">
+        <h4 class="list-group-item-heading">{{name}}</h4>
+    </div>
+
+    <div class="list-group-item">
+        <h4 class="list-group-item-heading">Information</h4>
+        <div class="list-group-item-text item-margin">Public access: <span id="keyvault.vaults.{{@key}}.public_access_allowed">{{ convert_bool_to_enabled public_access_allowed }}</span></div>
+    </div>
+
+</script>
+
+<script>
+    Handlebars.registerPartial("services.keyvault.vaults", $("#services\\.keyvault\\.vaults\\.partial").html());
+</script>
+
+<!-- Single Key Vault template -->
+<script id="single_keyvault_vaults-template" type="text/x-handlebars-template">
+    <div style="text-align: right; padding-right: 10px; text-weight: bold;"><a href="javascript:hidePopup()">X</a></div>
+    {{> services.keyvault.vaults}}
```

Comment:
I'm sorry what

Reply:
Allow me to remind you of this [pain in the rear](https://github.com/nccgroup/ScoutSuite/pull/132/files#diff-795b3bcacbbb37ba1939693c79cb20a7) of a pull request.

Reply:
Nice catch, wrong copy/paste... Sorry about that. :x

Reply:
Hahaha, I accept yout apology.

Reply:
Hahaha, I accept yout apology.